services/pull/merge.go

```diff
@@ -393,7 +393,12 @@ func getDiffTree(repoPath, baseBranch, headBranch string) (string, error) {
 	out := bytes.Buffer{}
 	scanner := bufio.NewScanner(strings.NewReader(list))
 	for scanner.Scan() {
-		fmt.Fprintf(&out, "/%s\n", scanner.Text())
+		filepath := scanner.Text()
+		if strings.HasPrefix(filepath, `"`) { // the filepath contains " or \.
+			filepath = strings.TrimPrefix(filepath, `"`)
```

Comment:
This doesn't seem to deal with `\`.

```
$ git diff-tree a06e7ac9fdbed88f73736b876ab63dfb680037ef aad2c7e2d2e381951cbaad3be107742f0b8fc132
:100644 100644 f1e195345db6ee7433b331c3aa333b50d46088f3 5c1b14949828006ed75a3e8858957f86a2f7e2eb M      "hello\\one\\\\two"
```

The actual file name is `helo\one\\two`

And there's also other edge cases like `"with\nnew\nlines"` or `"hello\ttabs"`.

Unless we want to print them that way, which could work.